drake/systems/plants/RigidBodyTree.cpp

```diff
@@ -226,9 +226,9 @@ map<string, int> RigidBodyTree::computePositionNameToIndexMap() const
   return name_to_index_map;
 }
 
-DrakeCollision::ElementId RigidBodyTree::addCollisionElement(const RigidBody::CollisionElement& element, const shared_ptr<RigidBody>& body, string group_name)
+DrakeCollision::ElementId RigidBodyTree::addCollisionElement(const RigidBody::CollisionElement& element, const shared_ptr<RigidBody>& body, const string& group_name)
 {
-  DrakeCollision::ElementId id(collision_model->addElement(element));
+  DrakeCollision::ElementId id = collision_model->addElement(element);
```

Comment:
Why would you use a copy constructor? It's similar to
```
MatrixXd H = massMatrix();
```
vs.
```
MatrixXd H(massMatrix());
```
The former is more idiomatic, and triggers return value optimization (hardly any overhead in this case, but in general), whereas I think an extra copy will be made if you explicitly call the copy constructor like that.